src/redux/configureStore.js

```diff
@@ -0,0 +1,14 @@
+import { createStore, combineReducers, applyMiddleware } from 'redux';
+import logger from 'redux-logger';
+import booksReducer from './books/books';
+
+const reducer = combineReducers({
+  booksReducer,
+});
```

Comment:
- [ ] It will be nice to also Imports `categoriesReducer` from `src/redux/categories`

- [ ] `booksReducer` and `categoriesReducer` should be added to `combineReducers`. Currently, we only have the `booksReducer`. It will be nice to fix this issue.